test/e2e_node/util.go

```diff
@@ -43,6 +43,7 @@ func Retry(maxWait time.Duration, wait time.Duration, retry RetryFn) []error {
 		} else {
 			return []error{}
 		}
+		time.Sleep(wait)
```

Comment:
I actually want to delete this entirely and just use Eventually instead.  I'll try that